packages/cloudflare/src/cli/commands/preview.ts

```diff
@@ -25,10 +27,11 @@ export async function previewCommand(args: WithWranglerArgs<{ cacheChunkSize: nu
 	const wranglerConfig = readWranglerConfig(args);
 
 	await populateCache(options, config, wranglerConfig, {
-		target: "local",
+		target: args.remote ? "remote" : "local",
 		environment: args.env,
 		wranglerConfigPath: args.wranglerConfigPath,
 		cacheChunkSize: args.cacheChunkSize,
+		shouldUsePreviewNamespace: args.remote,
```

Comment:
Should `shouldUsePreviewNamespace` just be `remote` / `useRemote`

Reply:
I don't think so. Both the preview and deploy commands can populate remote, but only the preview command wants to use preview namespaces when deploying remote.